python/pyiceberg/utils/datetime.py

```diff
@@ -91,6 +91,25 @@ def timestamp_to_micros(timestamp_str: str) -> int:
     raise ValueError(f"Invalid timestamp without zone: {timestamp_str} (must be ISO-8601)")
 
 
+def datetime_to_millis(dt: datetime) -> int:
+    """Converts a datetime to milliseconds from 1970-01-01T00:00:00.000000."""
+    if dt.tzinfo:
+        delta = dt - EPOCH_TIMESTAMPTZ
+    else:
+        delta = dt - EPOCH_TIMESTAMP
+    return (delta.days * 86400 + delta.seconds) * 1_000 + delta.microseconds // 1_000
+
+
+def timestamp_to_millis(timestamp_str: str) -> int:
```

Comment:
Why add this one when we don't use it? I would rather leave it out.

Reply:
Okay, I'll leave out this